src/DefinitionResolver.php

```diff
@@ -89,7 +89,11 @@ public function getDocumentationFromNode(Node $node)
         } else {
             $docBlock = $node->getAttribute('docBlock');
             if ($docBlock !== null) {
-                return $docBlock->getSummary();
+                if (empty($docBlock->getDescription()->render())) {
```

Comment:
I would have assumed `getDescription()` to return `null`, is that not the case?
the getter and `render()` should only be called once.